pkg/cloudprovider/providers/photon/photon.go

```diff
@@ -210,7 +215,7 @@ func getVMIDbyIP(pc *PCCloud, IPAddress string) (string, error) {
 func getPhotonClient(pc *PCCloud) (*photon.Client, error) {
 	var err error
 	if len(pc.cfg.Global.CloudTarget) == 0 {
-		return nil, fmt.Errorf("Photon Controller endpoint was not specified.")
+		return nil, fmt.Errorf("Photon controller endpoint was not specified")
```

Comment:
I think this needs to remain as `Controller`.

https://github.com/kubernetes/kubernetes/search?q=%22Photon+Controller%22&unscoped_q=%22Photon+Controller%22